pkg/controllers/provisioning/v1alpha1/allocation/filter.go

```diff
@@ -46,11 +46,11 @@ func (f *Filter) GetProvisionablePods(ctx context.Context, provisioner *v1alpha1
 	provisionable := []*v1.Pod{}
 	for _, pod := range pods.Items {
 		if err := functional.ValidateAll(
-			func() error { return f.isUnschedulable(&pod) },
-			func() error { return f.matchesProvisioner(&pod, provisioner) },
-			func() error { return f.hasSupportedSchedulingConstraints(&pod) },
-			func() error { return f.toleratesTaints(&pod, provisioner) },
-			func() error { return f.hasSupportedLabels(&pod) },
+			func() error { return f.isUnschedulable(ctx, &pod) },
```

Comment:
Do you need to add ctx to all these? Looks like the only one that uses it is `withValidConstraints`

Reply:
Done